ui/app/components/version-info/index.hbs

```diff
@@ -0,0 +1,7 @@
+<div class="version-info">
+  <ul class="version-info-list">
+    <li>Waypoint {{this.versionInfo.version}}</li>
+    <li>API version: {{this.versionInfo.api.current}}</li>
```

Comment:
Do you think we should maybe make these versions clickable to the docs eg. the protocol version info section?

Reply:
I think adding links would help give that data some context too.

Also wondering if we can shorten the API and entry point versions labels a bit - could we make them `API v1` and `Entrypoint v1`?

Reply:
Other option I considered: styling them in a `warning`/`error` color if minimum > current . Thoughts on that @dizzyup ?

Reply:
Oh interesting! Perhaps if we're doing that check we could return an additional badge instead - would sit inline next to the current version and say "Update available".

Reply:
Added: 
<img width="1142" alt="Screen Shot 2020-12-08 at 16 49 59" src="https://user-images.githubusercontent.com/1416421/101506250-757cc480-3975-11eb-91d6-daee05beb802.png">

Reply:
@dizzyup labels and version numbers are updated too